### PR TITLE
Unit tests for the PCHub

### PR DESCRIPTION
**Pull request by daisyab**
These are unit tests for PCHub written using the rspec gem.

### CODE REVIEW

**Comment by nidhimj22** [comment]
please correct the indentation.

File: pc-hub/spec/controllers/users_controller_spec.rb [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+				expect(User.count).to eq(0)
+  		end
+
+			it 'redirects to the "new" view' do
+				post :create, user: attributes_for(:user, password: 'invalid')
+				expect(response).to render_template :new
+			end
+		end
+
+  #context 'when password is valid' do
```
Comment (nidhimj22) [inline pc-hub/spec/controllers/users_controller_spec.rb]: why is this case commented?

**Comment by nidhimj22** [comment]
please correct the indentation.

**Comment by nidhimj22** [comment]
please document all the cases like. 
create new user - valid, invalid. 
sign in - valid credentials, invalid credentials.

**Comment by nidhimj22** [comment]
@rosariorobinson @izacristina  Can you review and merge this?

**Comment by rosariorobinson** [comment]
Is this PR okay to merge?

**Pull request merged**